src/backends/beamcoder.ts

```diff
@@ -55,65 +54,6 @@ const createDecoder = ({
     });
 };
 
-/**
- * A filter to convert between color spaces.
- * An example would be YUV to RGB, for mp4 to png conversion.
- */
-const createFilter = async({
-    stream,
-    outputPixelFormat,
-    interpolateFps,
-    interpolateMode = 'fast',
-}: {
-    stream: beamcoder.Stream;
-    outputPixelFormat: string;
-    interpolateFps?: number;
-    interpolateMode?: InterpolateMode;
-}): Promise<beamcoder.Filterer> => {
-    if (!stream.codecpar.format) {
-        return null;
-    }
-
-    let filterSpec = [`[in0:v]format=${stream.codecpar.format}`];
-
-    if (interpolateFps) {
-        if (interpolateMode === 'high-quality') {
-            filterSpec = [...filterSpec, `minterpolate=fps=${interpolateFps}`];
-        }
-        else if (interpolateMode === 'fast') {
-            filterSpec = [...filterSpec, `fps=${interpolateFps}`];
-        }
-        else {
-            throw new Error(`Unexpected interpolation mode: ${interpolateMode}`);
-        }
-    }
-
-    const filterSpecStr = filterSpec.join(', ') + '[out0:v]';
-
-    VERBOSE && console.log(`filterSpec: ${filterSpecStr}`);
-
-    return beamcoder.filterer({
-        filterType: 'video',
-        inputParams: [
-            {
-                name: 'in0:v',
-                width: stream.codecpar.width,
-                height: stream.codecpar.height,
-                pixelFormat: stream.codecpar.format,
-                timeBase: stream.time_base,
-                pixelAspect: stream.sample_aspect_ratio,
-            },
-        ],
-        outputParams: [
-            {
-                name: 'out0:v',
-                pixelFormat: outputPixelFormat,
-            },
-        ],
-        filterSpec: filterSpecStr,
-    });
```

Comment:
even with equa input pixelFormat and outputPixelFormat with filter adds and overhead